### ISSUE TITLE
Questions regarding the supported models.

### ISSUE THREAD

**Issue by mrbeann** [Accelergy-Project/pytorch2timeloop-converter#3]
Hi! Firstly, I want to thank you for your awesome work. 

As a beginner in Accelergy, I got into problems in creating yaml files for different NN models.

In the documentation I saw it states this tool can support "certain transformers" but it seems it only supports Conv and linear layers. Did I miss something?
Besides, is it possible to implement many other layers in Pytorch, e.g., normalization layer, recurrent layer, pooling layer?

**Comment by gilbertmike** [comment]
I know the issue was opened a long time ago.

This project recently got a major update which added support for many other layers and `torch.nn.functional` functions. You can take a look at the testing suite for examples.

To respond to the layers you mentioned, pooling layers are supported, normalization layers is not often modeled in Timeloop so not supported (although support can easily be added if it's needed), and I'm not sure about recurrent layers, it depends on how the layers are implemented (operations like tensor products and elementwise additions are supported so layers like LSTM could be constructed with those).

Since this is old, I'll close the issue for now. But I'm happy to help, so please let me know if you have more questions and I'll reopen the issue.
